### DISCUSSION TITLE
Import existing resource to the stack

### DISCUSSION THREAD

**Discussion by throrin19** [serverless/serverless#11842] [Q&A]
Is it possible to declare an already existing resource in the resource part of the yml file?

In my case, I want to be able to create an SQS queue in my serverless stack. The problem is that, for some courses, the queue in question already exists, it was created manually. Since its use is "critical for us, I can't delete it from AWS and then recreate it via serverless.

That's why I thought I would try to see with imported resources: Import the resource if it exists in the stack, otherwise create it.

Is this feasible? If yes, how?

**Comment by gaperSmile** [comment]
I have the same problem here!

We want to migrate a AWS infrastructure created manualy to IAC using Serverless Framework.
But I don't konw how to migrate the resources to my Serverless project without destroy it!

Is it just impossible or have I miss simething on the documentation?

**Comment by somesh-banerjee** [comment]
You can import it manually into the cloud formation stack that the serverless has created.

**Reply by throrin19** [reply]
AWS interface for this is really complicated. It is not possible using serverless params directly ?
